huxley/api/permissions.py

```diff
@@ -222,6 +222,47 @@ def has_permission(self, request, view):
         return user_is_advisor(request, view, school_id)
 
 
+class CommitteeFeedbackListPermission(permissions.BasePermission):
+    '''Accept GET for only the chair of the committee'''
+
+    def has_permission(self, request, view):
+        user = request.user
+        if user.is_superuser:
+            return True
+
+        method = request.method
+        committee_id = request.query_params.get('committee_id', -1)
+        return method == 'GET' and user_is_chair(request, view, committee_id)
+
+
+class CommitteeFeedbackDetailPermission(permissions.BasePermission):
+    '''Accept POST for only the delegate of the committee
+       Accept GET request from chair of committee'''
+
+    def has_permission(self, request, view):
+        user = request.user
+        if user.is_superuser:
+            return True
+
+        method = request.method
+        committee_id = request.data.get('committee', -1)
+        feedback_id = view.kwargs.get('pk', None)
```

Comment:
It's probably better to have this as -1 instead of None

Reply:
Done